addons/portal/views/portal_templates.xml

```diff
@@ -448,8 +488,9 @@
 
     <template id="portal_my_security">
         <t t-call="portal.portal_layout"><div class="o_portal_security_body">
-            <t t-set="additional_title">Security</t>
-            <t t-set="no_breadcrumbs" t-value="1"/>
+            <t t-set="page_title" t-value="'Password &amp; Security'"/>
```

Comment:
Same remark (not translated as this is a string): either use ``<t t-set="page_title">Password</t>``, either controller